### PR TITLE
First draft of Personal Applications

### PR DESCRIPTION
**Pull request by SherinDaher-Runai**

### CODE REVIEW

File: docs/Researcher/best-practices/personal-applications.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Personal Applications
+
+This article explains the procedure to create personal applications.
+
+Applications are used for API integrations with Run:ai. An application contains a client secret. Using the client key you can obtain a token and use it within subsequent API calls.
```
Comment (ozRunAI) [inline docs/Researcher/best-practices/personal-applications.md:5]:
```suggestion
Applications are used for API integrations with Run:ai. An application contains a client ID and a client secret. Using the client credentials you can obtain a token and use it within subsequent API calls.
```

File: docs/admin/authentication/applications.md [hunk 2]
```diff
@@ -1,32 +1,36 @@
-This article explains the procedure to manage applications and it’s permissions.
+This article explains the procedure to manage applications.
 
-Applications are used for API integrations with Run:ai. An application contains a secret key. Using the secret key you can obtain a token and use it within subsequent API calls.
+Applications are used for API integrations with Run:ai. An application contains a client secret. With the client secret you can obtain a token and use it within subsequent API calls.
```
Comment (ozRunAI) [inline docs/admin/authentication/applications.md:3]:
```suggestion
Applications are used for API integrations with Run:ai. An application contains a client key and secret. Using the client credentials you can obtain a token and use it within subsequent API calls.
```

File: docs/Researcher/best-practices/personal-applications.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Personal Applications
```
Comment (ozRunAI) [inline docs/Researcher/best-practices/personal-applications.md:1]: I'm not sure we would like to promote it in this name - Maybe User Applications? not sure

**Review by ozRunAI — CHANGES_REQUESTED** [review]

File: docs/Researcher/best-practices/user-applications.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# User Applications
+
+This article explains the procedure to create your applications.
+
+Applications are used for API integrations with Run:ai. An application contains a client ID and a client secret. With the client credentials you can obtain a token and use it within subsequent API calls.
```
Comment (ozRunAI) [inline docs/Researcher/best-practices/user-applications.md:5]: link to https://docs.run.ai/latest/developer/rest-auth/ ?

File: docs/Researcher/best-practices/user-applications.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Creating Personal Applications
```
Comment (ozRunAI) [inline docs/Researcher/best-practices/user-applications.md:8]:
```suggestion
## Creating Applications
```

**Review by ozRunAI — CHANGES_REQUESTED** [review]

File: docs/developer/rest-auth.md [hunk 6]
```diff
@@ -59,8 +46,8 @@ Replace `<COMPANY-URL>` below with:
         }
```
Comment (ozRunAI) [inline docs/developer/rest-auth.md:46]: reqUrl = "https://cp-590d-run-13764-kc-upgrade.runailabs.com/api/v1/token"
?!

File: docs/developer/rest-auth.md [hunk 7]
```diff
@@ -43,8 +30,8 @@ Replace `<COMPANY-URL>` below with:
           --header 'Content-Type: application/json' \
           --data-raw '{
           "grantType":"app_token",
```
[the remark this answers is not in the document]
Comment (ozRunAI) [reply docs/developer/rest-auth.md:32]: 'https://<runai_url>/api/v1/token' \

**Review by ozRunAI — CHANGES_REQUESTED** [review]

**Review by ozRunAI — CHANGES_REQUESTED** [review]

File: docs/developer/user-applications.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+# User Applications
+
+This article explains the procedure to create your own user applications.
+
+Applications are used for API integrations with Run:ai. An application contains a client ID and a client secret. With the client credentials, you can obtain a token as detailed in [API authentication](../developer/rest-auth.md) and use it within subsequent API calls.
+
+!!!Note
+    User applications are supported in cluster version 2.20 and above.
```
[the remark this answers is not in the document]
Comment (SherinDaher-Runai) [reply docs/developer/user-applications.md]: Keep note as is

File: docs/developer/user-applications.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+# User Applications
+
+This article explains the procedure to create your own user applications.
+
+Applications are used for API integrations with Run:ai. An application contains a client ID and a client secret. With the client credentials, you can obtain a token as detailed in [API authentication](../developer/rest-auth.md) and use it within subsequent API calls.
```
[the remark this answers is not in the document]
Comment (SherinDaher-Runai) [reply docs/developer/user-applications.md:5]: Updated

**Review by ozRunAI — APPROVED** [review]

**Pull request merged**